README.md

```diff
@@ -361,7 +361,15 @@ to set `true`; it's effectively a shortcut for `foo=true`).
 - `loops` -- optimizations for `do`, `while` and `for` loops when we can
   statically determine the condition
 
-- `unused` -- drop unreferenced functions and variables
+- `unused` -- drop unreferenced functions and variables (simple direct variable
+  assignments do not count as references unless set to `"keep_assign"`)
+
+- `toplevel` -- drop unreferenced functions (`"funcs"`) and/or variables (`"vars"`)
+  in the toplevel scope (disabled by default)
+
+- `top_retain` -- an array of names to prevent specific toplevel functions
+  and variables from removal even if unreferenced (has no effect when `toplevel`
+  is `false`)
```

Comment:
`top_retain` can also accept an array of strings, same way as `pure_funcs`

I guess I could feed all the top-level variables to the `RegExp` (or even a `function()`) and get it done that way.

Ah, yes, I forgot the implicit thing - makes sense since if you use `top_retain` you should have a good overview on the global variables by then.